README.md

```diff
@@ -275,7 +313,7 @@ const executeEthModuleResultObj = JSON.parse(executeEthModuleResult.value);
 log.info("Ethereum network set up successfully");
 ```
 
-This is the same code we already executed in the REPL, cleaned up for Typescript. The only new bits to pay attention to are the error-checking: all `EnclaveContext` and `ModuleContext`  methods return [a Result object][neverthrow] (much like in Rust). If the is a non-`Ok` result, we'll throw the result which will cause Mocha to mark the test as failed.
+Pay attention to the error-checking: all `NetworkContext` methods, as well as the `Test.setup` and `Test.run` methods, return [a Result object][neverthrow] (much like in Rust). If `setup` or `run` return a non-`Ok` result, the test will be marked as failed. This allows for easy, consistent error-checking: simply propagate the error upwards.
```

Comment:
Is this from an old version of the file? We haven't had `Test.setup` and `Test.run` since we got rid of the testsuites ages ago.

Should probably be:
```
This code uses the Kurtosis SDK, which is the same SDK that the Kurtosis CLI uses to communicate with the Kurtosis engine. The only new bits to pay attention to are the error-checking: all `EnclaveContext` and `ModuleContext` methods return [a Result object][neverthrow] (much like in Rust). If `result.isErr()`, we'll throw the result which will cause Mocha to mark the test as failed.
```